.github/workflows/test.yml

```diff
@@ -3,10 +3,21 @@ name: test
 on: [push, pull_request]
 
 jobs:
+  ruby-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.versions.outputs.value }}
+    steps:
+      - id: versions
+        run: |
+          versions=$(curl -s 'https://cache.ruby-lang.org/pub/misc/ci_versions/cruby.json' | jq -c '. + ["2.4"]')
```

Comment:
2.5, 2.6 are not in this set, should they be?

Reply:
I don't know either, but the tests run quickly, and perhaps there's added feedback to found in the 2.5, 2.6 test runs? Well, it's up to you.

Reply:
There is no strong opinion that. I added them.